### PR TITLE
unittest for paddle by default

### PR DESCRIPTION
**Pull request by rical730**

### CODE REVIEW

File: .teamcity/windows_requirements_paddle.txt [hunk 1]
```diff
@@ -2,4 +2,4 @@
 gym
 details
 parameterized
-paddlepaddle==2.0.0
+paddlepaddle
```
Comment (TomorrowIsAnOtherDay) [inline .teamcity/windows_requirements_paddle.txt]: paddlepaddle>=2.0.0

File: CMakeLists.txt [hunk 2]
```diff
@@ -98,10 +98,10 @@ if (WITH_TESTING)
             endif()
         endforeach()
     else ()
-        file(GLOB_RECURSE TEST_OPS RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}" "*_test.py")
+        file(GLOB_RECURSE TEST_OPS RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}" "*_test*.py")
```
Comment (TomorrowIsAnOtherDay) [inline CMakeLists.txt]: Please run paddle tests separately according to the discussion.

Comment (rical730) [reply CMakeLists.txt]: done

File: .teamcity/windows_requirements_paddle.txt (hunk 1 above)
Comment (rical730) [reply .teamcity/windows_requirements_paddle.txt]: done

File: CMakeLists.txt [hunk 3]
```diff
@@ -105,5 +105,10 @@ if (WITH_TESTING)
                 py_test(${src} SRCS ${src}.py ENVS ${PADDLE_PYTHON_PATH})
             endif()
         endforeach()
+        file(GLOB_RECURSE TEST_OPS RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}" "*_test_paddle.py")
```
Comment (zenghsh3) [inline CMakeLists.txt:108]: Can we just naming the unit test of paddle with   `*_test.py`?

Comment (rical730) [reply CMakeLists.txt:108]: I think it is better to maintain a unified naming method with fluid and torch

Comment (zenghsh3) [reply CMakeLists.txt:108]: ok

**Review by zenghsh3 — DISMISSED** [review]
LGTM

**Review by TomorrowIsAnOtherDay — APPROVED** [review]
LGTM.

**Pull request merged**